src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.Client.fs

```diff
@@ -197,16 +201,30 @@ type Client<'TAttribute, 'TContext when 'TAttribute :> AnalyzerAttribute and 'TC
                         assembly.GetExportedTypes()
                         |> Seq.collect (Client.analyzersFromType<'TAttribute, 'TContext> path)
                         |> Seq.filter (fun registeredAnalyzer ->
-                            let shouldExclude = excludedAnalyzers.Contains(registeredAnalyzer.Name)
-
-                            if shouldExclude then
-                                logger.LogInformation(
-                                    "Excluding {0} from {1}",
-                                    registeredAnalyzer.Name,
-                                    assembly.FullName
-                                )
-
-                            not shouldExclude
+                            match excludeInclude with
+                            | Some(Exclude excluded) ->
+                                let shouldExclude = excluded.Contains(registeredAnalyzer.Name)
+
+                                if shouldExclude then
+                                    logger.LogInformation(
+                                        "Excluding {0} from {1}",
```

Comment:
fwiw, I think it's usually recomended to use named place holder values in logging message templates rather than numbers so you get better information when using structured logs (there is a [Roslyn analyzer](https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca2253) that detects that)

Reply:
Thanks!